From ebbded98bb17ae48a0a48065dd458eed2161c124 Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Mon, 21 Nov 2011 20:18:47 +0100
Subject: [PATCH 19/63] carl9170 firmware: remove wol #ifdef leftovers

Thanks to the aggressive cflags, the compiler will
optimize the check away if wol is not enabled.

Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
---
 carlfw/src/wlan.c |    2 --
 1 file changed, 2 deletions(-)

--- a/carlfw/src/wlan.c
+++ b/carlfw/src/wlan.c
@@ -707,12 +707,10 @@ static unsigned int wlan_rx_filter(struc
 		rx_filter |= CARL9170_RX_FILTER_MGMT;
 	}
 
-#ifdef CONFIG_CARL9170FW_WOL
 	if (unlikely(fw.suspend_mode == CARL9170_HOST_SUSPENDED)) {
 		wol_rx(rx_filter, hdr, min(data_len,
 			(unsigned int)AR9170_BLOCK_SIZE));
 	}
-#endif /* CONFIG_CARL9170FW_WOL */
 
 #undef AR9170_RX_ERROR_BAD
 
